views/beneficial-owner-other.html

```diff
@@ -35,16 +36,29 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">Tell us about the corporate benefi
         }) }}
         {% call govukFieldset({
           legend: {
-            text: "What is their principal office address?",
+            text: "What is its principal or registered office address?",
             classes: "govuk-fieldset__legend--m",
             isPageHeading: false
           }
         }) %}
 
         {{ govukInput({
           label: {
-            html: 'Building and street <span class="govuk-visually-hidden">line 1 of 2</span>'
+            html: 'Property name or number (optional)'
+          },
+          hint: {
+            text: "For example, 'Saron House', '116' or 'Unit 37a'"
+          },
+          id: "principalAddressPropertyNameNumber",
```

Comment:
Ids and others should match the data model after the changes

Reply:
Kept as serviceAdress just changed the case of both sets of field names